recipes/bar_chart_race/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "bar_chart_race" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 70c0f6f897a4746472123e509431e75e75a13d5613c27ef8c5b90c8fabdb80b6
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - pandas
+    - numpy
+    - matplotlib
+
+test:
+  imports:
+    - bar_chart_race
+
+about:
+  home: https://github.com/dexplo/bar_chart_race
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
Add a comment with the following link as a reminder to check if the license file is included in the source distribution in future releases:

https://github.com/dexplo/bar_chart_race/issues/2